### DISCUSSION TITLE
Compute Hours - how exactly is the charging done

### DISCUSSION THREAD

**Discussion by Aging-Developer** [supabase/supabase#20583] [Questions] [answered]
Hi,   How does Compute by the Hourly actually work?

i.e. if i had a cron job that runs every hour for 5 seconds and zero other activity, is that "was used every hour, so will be charged for every hour", or is it "5secs *24 hours = 2 mins, therefore 30 days before using a full hour"?

Just example to try to explain my question a bit more!

**Comment by GaryAustin1** [comment, marked as answer]
Every hour your instance is running.  It has nothing to do with what the instance is doing.  You have full CPU for the tier at all times.
